### PR TITLE
Add Ubuntu 22.10 to ci

### PR DESCRIPTION
**Pull request by 0xxon**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -213,6 +213,18 @@ opensuse_tumbleweed_task:
...
+  container:
+    # Ubuntu 22.10 EOL: July 2023
+    dockerfile: ci/ubuntu-22.10/Dockerfile
+    << : *RESOURCES_TEMPLATE
+  << : *CI_TEMPLATE
+  env:
+    ZEEK_CI_CREATE_ARTIFACT: 1
+  upload_binary_artifacts:
+    path: build.tgz
+  benchmark_script: ./ci/benchmark.sh
```
Comment (ckreibich) [inline .cirrus.yml:226]: Do we want to duplicate the benchmarking from `ubuntu22_task`? I'm not sure what's guided this in the past — @timwoj might know best.

Comment (timwoj) [reply .cirrus.yml:226]: No, it should only be in the ubuntu22 task.

Comment (0xxon) [reply .cirrus.yml:226]: ups, sorry about that.

**Comment by timwoj** [comment]
Do you want to squash this before I merge it?

**Comment by 0xxon** [comment]
Sure, let me quickly do that...

**Pull request merged**